pkg/netpol/connlist/connlist.go

```diff
@@ -204,15 +203,9 @@ func (ca *ConnlistAnalyzer) ConnlistFromK8sCluster(clientset *kubernetes.Clients
 	return ca.getConnectionsList(pe)
 }
 
-// ConnectionsListToString returns a string of connections from list of Peer2PeerConnection objects
-func (ca *ConnlistAnalyzer) ConnectionsListToString(conns []Peer2PeerConnection) string {
-	connLines := make([]string, len(conns))
-	for i := range conns {
-		connLines[i] = conns[i].String()
-	}
-	sort.Strings(connLines)
-	newlineChar := fmt.Sprintln("")
-	return strings.Join(connLines, newlineChar)
+// ConnectionsListToString returns a string of connections from list of Peer2PeerConnection objects in the required output format
+func (ca *ConnlistAnalyzer) ConnectionsListToString(conns []Peer2PeerConnection, outputFormat ConnsFormatter) (string, error) {
```

Comment:
`outputFormat` should be a member of `ConnlistAnalyzer` instead of argument to this function.
Should use the type of `ConnlistAnalyzerOption` to add a new option as function `WithOutputFormat`.